### PR TITLE
OSP-175: update bosi for queens deployment

### PR DESCRIPTION
**Pull request by wolverineav**
Reviewer: @sarath-kumar @WeifanFu-bsn 

 - since we split packages in multiple parts, bosi needs to handle all of those
 - also removed all `ubuntu` bash scripts that are never used, but remain for historical reasons
 - `ubuntu` scripts do not work with any known orchestrator, keeping them only increases heft
   in our code

### CODE REVIEW

File: etc/t6/puppet_template/centos_controller.pp [hunk 1]
```diff
@@ -17,6 +17,38 @@
...
 }
 
+# lldp
+file { "/bin/bsnlldp":
+    ensure  => file,
+    mode    => "0775",
+}
+file { "/usr/lib/systemd/system/bsnlldp.service":
+    ensure  => file,
+    content => "
```
Comment (WeifanFu-bsn) [inline etc/t6/puppet_template/centos_controller.pp]: Is the single double quote correct?

**Comment by WeifanFu-bsn** [comment]
You mentioned that ubuntu scripts are deleted as they are not used. But what about those pp files? (For example, redhat_compute.pp is also deleted.)

It is probably a legit change, but I am a bit curious what those files were for since it was not mentioned in the PR.

Other changes LGTM.

**Comment by wolverineav** [comment]
> (For example, redhat_compute.pp is also deleted.)

ah, forgot to mention that - `redhat_compute.pp` is same as `ubuntu` - we don't support via bosi.
it is done via RHOSP integration. previously we used to deploy our plugin to redhat using bosi. not the case anymore.

we do support sriov deployment - which remains as part of code even now.

**Comment by wolverineav** [comment]
merging this for testing by QA. will be cherry-picked to stable branch after testing

**Pull request merged**